src/components/ActivityButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@mui/material';
+
+type ActivityButtonProps = {
+    color?: "inherit" | "primary" | "secondary";
+    handleClick?: () => void;
```

Comment:
Would suggest to rename this to `onClick`.
`handleClick` is the actual handler of the click event, which can be tied to the onClick event itself:
```tsx
<ActivityButton onClick={() => return "I am the click handler!"}/>
```

Makes it easier to read a bit as well.

Reply:
Changed it to 'onClick', and made it to not optional (just forgot to remove it 😅)